main.rb

```diff
@@ -0,0 +1,80 @@
+require_relative 'app'
+
+class LibraryApp
+  def initialize
+    @app = App.new
+  end
+
+  def start
+    display_welcome_message
+
+    loop do
+      display_menu
+      option = user_input
+      handle_option(option)
+      break if option == 7
+    end
+  end
+
+  private
+
+  def display_welcome_message
+    puts 'Welcome to School Library App!'
+    puts ' '
+  end
+
+  def display_menu
+    puts 'Please select an option by entering a number:'
+    puts '1. List all books'
+    puts '2. List all people'
+    puts '3. Create a person'
+    puts '4. Create a book'
+    puts '5. Create a rental'
+    puts '6. List all rentals for a given person id '
+    puts '7. Exit'
```

Comment:
- [ ] Dear @NoelLincoln, You did great with adding and listing books, also adding rental seems to work fine. The main issue I face is listing all the rentals once created. The app raises an issue when I use the ID or the number of the person. Please make sure to overcome this issue and list all the rentals.

### The issue 
<img width="1136" alt="Screenshot 2023-11-25 at 9 35 11 at night" src="https://github.com/NoelLincoln/ruby_school_library/assets/69077061/4d4ea8ac-53f1-4c2c-8eea-d8ab0b1e6357">